<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossplotViewer</class>
 <widget class="QMainWindow" name="CrossplotViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/images/logo_fs_64x64.png</normaloff>:/icons/images/logo_fs_64x64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="RulerGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;Scale</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Fit_Window"/>
   </widget>
   <widget class="QMenu" name="menu_Display">
    <property name="title">
     <string>&amp;Display</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDisplay_Options"/>
    <addaction name="actionAttribute_Colortable"/>
    <addaction name="actionAttribute_Range"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_Trend_Line"/>
   </widget>
   <widget class="QMenu" name="menu_Broadcast">
    <property name="title">
     <string>&amp;Broadcast</string>
    </property>
    <addaction name="action_Receive_CDPs"/>
    <addaction name="action_Dispatch_CDPs"/>
   </widget>
   <widget class="QMenu" name="menuTrenss">
    <property name="title">
     <string>&amp;Trend</string>
    </property>
    <widget class="QMenu" name="menu_Compute_Trend">
     <property name="title">
      <string>&amp;Compute Trend</string>
     </property>
     <addaction name="actionCompute_Trend_From_Loaded_Data"/>
     <addaction name="actionCompute_Trend_From_Displayed_Data"/>
     <addaction name="actionCompute_Trend_From_Selected_Data"/>
    </widget>
    <addaction name="menu_Compute_Trend"/>
    <addaction name="action_Pick_Trend"/>
    <addaction name="actionSet_Angle"/>
    <addaction name="separator"/>
    <addaction name="action_Flatten_Trend"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <widget class="QMenu" name="menuHistogram">
     <property name="title">
      <string>Histogram</string>
     </property>
     <addaction name="action_HistogramXAxis"/>
     <addaction name="action_HistogramYAxis"/>
     <addaction name="action_HistogramAttribute"/>
    </widget>
    <addaction name="actionSelect_By_Inline_Crossline_Ranges"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuHistogram"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Data"/>
   <addaction name="menuTrenss"/>
   <addaction name="menu_Display"/>
   <addaction name="menu_Broadcast"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_By_Inline_Crossline_Ranges"/>
   <addaction name="separator"/>
   <addaction name="actionDisplay_Options"/>
   <addaction name="actionAttribute_Range"/>
   <addaction name="actionAttribute_Colortable"/>
   <addaction name="separator"/>
   <addaction name="action_HistogramXAxis"/>
   <addaction name="action_HistogramYAxis"/>
   <addaction name="action_HistogramAttribute"/>
   <addaction name="separator"/>
   <addaction name="action_Receive_CDPs"/>
   <addaction name="action_Dispatch_CDPs"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Zoom-Plus-icon.png</normaloff>:/icons/Zoom-Plus-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Zoom-Minus-icon.png</normaloff>:/icons/Zoom-Minus-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_Fit_Window">
   <property name="text">
    <string>Zoom &amp;Fit Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionSelect_Data_by_Geometry">
   <property name="text">
    <string>Select Inline and Crossline &amp;Ranges</string>
   </property>
  </action>
  <action name="actionDisplay_Options">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/display-options-icon-32x32.png</normaloff>:/icons/images/display-options-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Display &amp;Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Receive_CDPs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/receive-icon-32x32.png</normaloff>:/icons/images/receive-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Receive CDPs</string>
   </property>
  </action>
  <action name="action_Dispatch_CDPs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/send-icon-32x32.png</normaloff>:/icons/images/send-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Dispatch CDPs</string>
   </property>
  </action>
  <action name="action_Pick_Trend">
   <property name="text">
    <string>&amp;Pick Trend</string>
   </property>
  </action>
  <action name="action_Flatten_Trend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Flatten Trend</string>
   </property>
  </action>
  <action name="actionSelect_By_Inline_Crossline_Ranges">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/select-range2-icon-32x32.png</normaloff>:/icons/images/select-range2-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Data Range</string>
   </property>
  </action>
  <action name="actionDisplay_Trend_Line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;Trend Line</string>
   </property>
  </action>
  <action name="actionCompute_Trend_From_Loaded_Data">
   <property name="text">
    <string>&amp;From Loaded Data</string>
   </property>
  </action>
  <action name="actionCompute_Trend_From_Displayed_Data">
   <property name="text">
    <string>From &amp;Displayed Data</string>
   </property>
  </action>
  <action name="actionCompute_Trend_From_Selected_Data">
   <property name="text">
    <string>From &amp;Selected Data</string>
   </property>
  </action>
  <action name="actionAttribute_Colortable">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/colortable-icon-32x32.png</normaloff>:/icons/images/colortable-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Edit  &amp;Colortable</string>
   </property>
  </action>
  <action name="actionAttribute_Range">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/select-range-icon-32x32.png</normaloff>:/icons/images/select-range-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Attribute &amp;Range</string>
   </property>
  </action>
  <action name="actionSet_Angle">
   <property name="text">
    <string>&amp;Edit  Angle</string>
   </property>
  </action>
  <action name="action_HistogramXAxis">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/histogram-x-axis-icon-32x32.png</normaloff>:/icons/images/histogram-x-axis-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>x-axis</string>
   </property>
  </action>
  <action name="action_HistogramYAxis">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/histogram-y-axis-icon-32x32.png</normaloff>:/icons/images/histogram-y-axis-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>y-axis</string>
   </property>
  </action>
  <action name="action_HistogramAttribute">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/histogram-points-icon-32x32.png</normaloff>:/icons/images/histogram-points-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>attribute</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RulerGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>rulergraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>CrossplotViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
